domains/steeldev.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "steeldev0",
+           "email": "xrgaming272@gmail.com",
+           "discord": "976208121825460224"
+        },
+    
+        "record": {
+            "A": ["69.197.135.202"]
```

Comment:
Since this is a DBH proxy IP, please give a preview of your website.

Reply:
oh, this is the website pnode2.danbot.host:2951

Reply:
> oh, this is the website pnode2.danbot.host:2951

Would like you to add more stuff to your website